src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java

```diff
@@ -70,6 +72,8 @@
   private final boolean fetchLoanType;
 
   private static final String ITEMS_COLLECTION_PROPERTY_NAME = "items";
+  private static final String HOLDINGS_RECORDS_COLLECTION_PROPERTY_NAME = "holdingsRecords";
+  private static final String INSTANCES_RECORDS_PROPERTY_NAME = "instances";
```

Comment:
INSTANCES_RECORDS_COLLECTION_PROPERTY_NAME?
Please check 100 char limit after renaming

Reply:
just wanted to replace records to collection, fixed